src/fides/api/ops/util/connection_type.py

```diff
@@ -141,4 +142,23 @@ def is_match(elem: str) -> bool:
                 for item in manual_types
             ]
         )
+
+    if system_type == SystemType.email or system_type is None:
+        email_types: list[str] = sorted(
+            [
+                email_type.value
+                for email_type in ConnectionType
+                if email_type == ConnectionType.sovrn and is_match(email_type.value)
+            ]
+        )
+        connection_system_types.extend(
+            [
+                ConnectionSystemTypeMap(
+                    identifier=item,
+                    type=SystemType.email,
+                    human_readable=ConnectionType(item).human_readable,
+                )
+                for item in email_types
+            ]
+        )
```

Comment:
I added a new "email" connector type while I was here.  It's not a saas connector and it's not a db connector - it seemed like we needed a new type.